standard/ERCs/erc-6900.md

```diff
@@ -48,23 +41,15 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 - A **modular account** (or **modular smart contract account, MSCA**) is an account that supports modular functions. There are three types of modular functions:
-  - **Validation functions** validate the caller's authenticity and authority to the account.
-  - **Execution functions** execute any custom logic allowed by the account.
+  - **Validation functions** validate authorization on behalf of the account.
+  - **Execution functions** execute custom logic allowed by the account.
   - **Hooks** execute custom logic and checks before and/or after an execution function or validation function.
-- A **validation function** is a function that validates authentication and authorization of a caller to the account. There are two types of validation functions:
-  - **User Operation Validation** functions handle calls to `validateUserOp` and check the validity of an ERC-4337 user operation.
-  - **Runtime Validation** functions run before an execution function when not called via a user operation, and enforce checks. Common checks include allowing execution only by an owner.
-- An **execution function** is a smart contract function that defines the main execution step of a function for a modular account.
-- The **standard execute** functions are two specific execute functions that are implemented natively by the modular account, and not on a module. These allow for open-ended execution.
-- A **hook** is a smart contract function executed before or after another function, with the ability to modify state or cause the entire call to revert. There are four types of hooks:
-  - **Pre User Operation Validation Hook** functions run before user operation validation functions. These can enforce permissions on what actions a validation function may perform via user operations.
-  - **Pre Runtime Validation Hook** functions run before runtime validation functions. These can enforce permissions on what actions a validation function may perform via direct calls.
-  - **Pre Execution Hook** functions run before an execution function. They may optionally return data to be consumed by their related post execution hook functions.
-  - **Post Execution Hook** functions run after an execution function. They may optionally take returned data from their related pre execution hook functions.
-- An **associated function** refers to either a validation function or a hook.
-- A **native function** refers to a function implemented natively by the modular account, as opposed to a function added by a module.
-- A **module** is a deployed smart contract that hosts any amount of the above three kinds of modular functions: execution functions, validation functions, or hooks.
-- A module **manifest** is responsible for describing the execution functions, validation functions, and hooks that will be configured on the MSCA during installation, as well as the module’s metadata, dependency requirements, and permissions.
+There are two types of hooks:
```

Comment:
Do we also need to call out permission hooks here? Or is that what @howydev is working on? Could be confusing to mention "permissions" in line 48.

Reply:
I really think we should stop calling them "permission hooks". They're just validation-associated "execution hooks", and we already have well defined terminology for execution hooks.

Even in just the RI, there are many things that are logically "permissions" that aren't enforced via "permission hooks", such as the allowlist module.